.github/CONTRIBUTING.md

```diff
@@ -1,7 +1,15 @@
 # Contributing
 
-When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change.
+Thank you for your interest in contributing to our project! We welcome all contributions, whether it's a bug fix, feature suggestion, or documentation improvement.
+
+Before you start contributing, please take a moment to read the following guidelines:
+
+1. Initiate a discussion with the repository owners, this can be done through an `Github Issue`. [Issue_Template](https://github.com/scorelab/Go-social/blob/14e5dfcdfb59888efc318bb4835e3577d5d09532/.github/ISSUE_TEMPLATE.md).
+2. Please ensure that you discuss the change you intend to make before proceeding.
```

Comment:
No need as beginning we mention this.

Reply:
I have gone through all the changes you suggested and arrive at this final outcome

Reply:
# Contributing

Thank you for your interest in contributing to our project! We welcome all contributions, whether it's a bug fix, feature suggestion, or documentation improvement.

Before you start contributing, please take a moment to read the following guidelines:

1. Initiate a discussion with the repository owners, either by creating a `GitHub Issue` or using any other communication method, to discuss your intended changes before making any modifications. [`Issue_Template`](https://github.com/scorelab/Go-social/blob/14e5dfcdfb59888efc318bb4835e3577d5d09532/.github/ISSUE_TEMPLATE.md)
2. Maintain a `Fork` copy of the code base as a developer and it would be easy to contribute if the developer has a `Clone` copy in local development workspace.
3. Commit your changes with clear messages and open a `Pull Request` from your forked repository to the original one for contribution.   [`Pull_Request_Template`](https://github.com/scorelab/Go-social/blob/14e5dfcdfb59888efc318bb4835e3577d5d09532/.github/PULL_REQUEST_TEMPLATE.md)

Please note we have a code of conduct, please follow it in all your interactions with the project.

Reply:
Will this works?🤞

Reply:
What I feel is we do not need to mention template as it is already there when we are creating one

Reply:
Cool, i will remove them other than that is there any change.